src/python/grapl-common/grapl_common/envelope.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import uuid
+from uuid import uuid4
+
+from graplinc.grapl.api.services.v1beta1.types_pb2 import Envelope as _Envelope
+from graplinc.grapl.api.services.v1beta1.types_pb2 import Meta, ProtoUuidV4
```

Comment:
I like what you did above renaming the generated protobuf class to `_Envelope`, would also be nice to do it for `_Meta` and `_ProtoUuidV4`